### DISCUSSION TITLE
Xbox series x

### DISCUSSION THREAD

**Discussion by tomshale99** [Prince25/StockAlertBot#16] [Q&A] [answered]
I am new to coding and I am not sure how to run the stock alert bot in node. I have downloaded it and edited the main.js but I have no clue how to actually start the bot. Please could you help. (I am using it on Mac)

**Comment by tomshale99** [comment]
When I try to run it, I get this:

"https://www.smythstoys.com/uk/en-gb/video-games-and-tablets/xbox-gaming/xbox-series-x-%7c-s/xbox-series-x-%7c-s-consoles/xbox-series-x-1tb-console/p/192012"
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SyntaxError: Unexpected string
    at Loader.moduleStrategy (node:internal/modules/esm/translators:145:18)
    at async link (node:internal/modules/esm/module_job:47:21)

Is there a problem with the link I have used?

**Reply by tomshale99** [reply]
Ah actually I just realised it is not one of the supported stores

**Reply by Prince25** [reply]
Yeah, that store is not supported. Is that store a popular store in the UK? If so, we don't mind trying to add support for it.

**Reply by tomshale99** [reply]
Not particularly but it has been recently for xbox series x. Would definitely be worth trying to add stores like currys and game - potentially very.co.uk.They are all popular for electronics

**Reply by tomshale99** [reply]
https://www.currys.co.uk/gbuk/gaming/console-gaming/consoles/microsoft-xbox-series-x-1-tb-10203371-pdt.html

**Reply by Prince25** [reply]
Can you provide the other two links as well? Could you please rank them in order of which you'd like to see added first?

**Reply by tomshale99** [reply]
The most important two would be the currys one above and https://www.argos.co.uk/product/8448262 on argos (I know argos is supported but this link was not working before) - argos being the most important

**Reply by tomshale99** [reply]
i cannot find links for the other two as they do not allow you to view the page on the website

**Reply by Prince25** [reply]
I'm happy to report that @Jeisonx has implemented currys and we will be pushing that update soon. As for Argos, I've tried the link you provided for Xbox and it worked fine for me.

**Comment by Prince25** [comment, marked as answer]
> I am new to coding and I am not sure how to run the stock alert bot in node. I have downloaded it and edited the main.js but I have no clue how to actually start the bot. Please could you help. (I am using it on Mac)

Good day, Tom. Thanks for adding your question to the discussions. 

First, did you install [Node.js](https://nodejs.org/en/download/)? It doesn't matter if you do LTS or current. 
Second, edit the main.js to your liking as described in the [README](https://github.com/PrinceS25/StockAlertBot/blob/main/README.md).
Third, open up a [terminal](https://macpaw.com/how-to/use-terminal-on-mac#:~:text=To%20open%20it%2C%20either%20open,background%20open%20on%20your%20desktop.) in the main directory of the StockAlertBot, and type in `node main.js`.
